src/common/ast-path.js

```diff
@@ -20,6 +21,30 @@ class AstPath {
     this.stack = [value];
   }
 
+  get name() {
+    return getPenultimate(this.stack) || null;
+  }
+
+  get node() {
```

Comment:
Use `value`?

Reply:
This distinction value vs node is not practical. Node is a much better word here.

Reply:
If so, let's change remove old `.getValue()` change `.getNode()` to current `.getValue()`?

Reply:
`getNode` has a parameter. Also that might break plugins if, unlike Prettier, they happen to point the path to arrays. I thought we would just deprecate `getName` and `getValue`.

Reply:
Okay, never mind, I guess we'll stop using them. I try to change them to getters long time ago, that PR was maybe two years ago.

Reply:
Found it! https://github.com/prettier/prettier/pull/7505